### PR TITLE
ATWD-5263 [iOS][POC] Korean Keyboard POC

### PR DESCRIPTION
**Pull request by dkyurtov**

### CODE REVIEW

File: hangul/hangulkeyboard.c [hunk 1]
```diff
@@ -69,6 +69,8 @@
 
 #define HANGUL_KEYBOARD_TABLE_SIZE 0x80
 
+#define ASCII_TABLE_LEBGTH 127
```
Comment (rumen-cholakov) [inline hangul/hangulkeyboard.c]: There is a typo here `LEBGTH`

File: hangul/hangul.h
Comment (rumen-cholakov) [inline hangul/hangul.h:1]: Please add header comments with some information on the new functions you are adding

File: hangul/hangulctype.c [hunk 2]
```diff
@@ -73,6 +85,44 @@ hangul_is_choseong(ucschar c)
...
+    outbytesleft = bufsize;
+    ret = iconv(cd, &inbuf, &inbytesleft, &outbuf, &outbytesleft);
+    (void)ret;
+
+    iconv_close(cd);
+
+    if (outbytesleft > 0)
+	*outbuf = '\0';
+    else
+	buf[bufsize - 1] = '\0';
```
Comment (rumen-cholakov) [inline hangul/hangulctype.c:123]: I see that you've used the same formatting as the rest of the code but I would like to note that I'm not a fan of it and would much prefer if the new functions follow a more sane style guide.
If the rest of the team are ok with this I can accept it.

**Comment by rumen-cholakov** [comment]
@dkyurtov who has write access to this repository? I see that I don't have it 
image

File: hangul/hangulctype.c (hunk 2 above)
Comment (dkyurtov) [reply hangul/hangulctype.c:123]: I copied this function from the sample application [here](https://github.com/abaltatech/libhangul/blob/main/test/hangul.c). That is why I haven't changed the style in it.

**Review by rumen-cholakov — APPROVED** [review]

**Pull request closed without merging**